component/xload/utils.go

```diff
@@ -104,3 +108,13 @@ func RoundFloat(val float64, precision int) float64 {
 	ratio := math.Pow10(precision)
 	return math.Round(val*ratio) / ratio
 }
+
+func IsFilePresent(localPath string) (bool, int64) {
```

Comment:
IsFilePresent lets make this method private.

Reply:
Done